frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx

```diff
@@ -171,27 +172,55 @@ export const OngoingActivityBody: React.FC<OngoingActivityBodyProps> = ({
   } else {
     const allActivities = prometheusActivities.map(
       ({ results, loader, component: Component }, idx) => (
-        // eslint-disable-next-line react/no-array-index-key
-        <Activity key={idx}>
-          {loader ? (
-            <AsyncComponent loader={loader} results={results} />
-          ) : (
-            <Component results={results} />
+        <ErrorBoundary
+          // eslint-disable-next-line react/no-array-index-key
+          key={idx}
+          FallbackComponent={() => (
+            <Activity>
+              <Alert
+                variant="danger"
+                isInline
+                isPlain
+                title={t('console-shared~Extension error')}
+              />
+            </Activity>
           )}
```

Comment:
nit, DRY. Should we factor this out into a shared component that provides a little more context for inline errors caught by an ErrorBoundary? I believe ErrorBoundary passes some props down the FallbackComponent that we can use.

Reply:
Ah yeah -- I was hacking and then failed to reread that... I was a bad lazy dev heh.

Reply:
As for rendering errors -- I could look into something that will do what the full page one does, but inline is a real issue when it comes to printing potentially long-winded details. I was just looking for something better than an empty DIV which crashed the UI together if it was the only one in the feed.